<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMLServer2Class</class>
 <widget class="QWidget" name="XMLServer2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XMLServer2</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IPADDR</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>CLTNo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>MasterID</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Param</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
